packages/venia-ui/lib/components/ContactPage/contactPage.js

```diff
@@ -87,140 +93,169 @@ const ContactPage = props => {
+                                />
+                            </Field>
+                            <Field
                                 id="contact-comment"
-                                validate={isRequired}
-                                placeholder={formatMessage({
-                                    id: 'contactPage.commentPlaceholder',
-                                    defaultMessage: `Tell us what's on your mind`
+                                label={formatMessage({
+                                    id: 'contactPage.comment',
+                                    defaultMessage: 'Message'
                                 })}
-                            />
-                        </Field>
-                        <div className={classes.buttonsContainer}>
-                            <Button
-                                priority="high"
-                                type="submit"
-                                disabled={isBusy}
                             >
-                                <FormattedMessage
-                                    id={'contactPage.submit'}
-                                    defaultMessage={'Send'}
+                                <TextArea
+                                    autoComplete="comment"
+                                    field="comment"
+                                    id="contact-comment"
+                                    validate={isRequired}
+                                    placeholder={formatMessage({
+                                        id: 'contactPage.commentPlaceholder',
+                                        defaultMessage: `Tell us what's on your mind`
+                                    })}
                                 />
-                            </Button>
-                        </div>
-                    </Form>
-                </div>
-                <div className={classes.sideContent}>
-                    {/** Side content CMS Block **/}
+                            </Field>
+                            <div className={classes.buttonsContainer}>
+                                <Button
+                                    priority="high"
+                                    type="submit"
+                                    disabled={isBusy}
+                                >
+                                    <FormattedMessage
+                                        id={'contactPage.submit'}
+                                        defaultMessage={'Send'}
+                                    />
+                                </Button>
+                            </div>
+                        </Form>
+                    </div>
+                    {contactUsSidebar}
                 </div>
-            </div>
-        </div>
+            </article>
+        </Fragment>
     );
 };
 
 ContactPage.propTypes = {
     classes: shape({
-        root: string,
+        loadingContainer: string,
```

Comment:
It seems as though this should stay as `root`? Not seeing `loadingContainer` changed elsewhere

Reply:
I refreshed the classes list by their order of appearance in the file.
`root` is just below that. I can move it back at the top.